src/renderer/component/fileDownloadLink/view.jsx

```diff
@@ -27,12 +35,7 @@ class FileDownloadLink extends React.PureComponent {
     }
   }
 
-  checkAvailability(uri) {
```

Comment:
This isn't currently doing anything. The action is disabled so I figured we can just remove it until that is fixed.